source/features/tags-on-commits-list.gql

```diff
@@ -0,0 +1,39 @@
+query GetTagsOnCommit(
+	$owner: String!
+	$name: String!
+	$commit: String!
+	$after: String
```

Comment:
Needs to be tested, may be undefined

Reply:
Great! It works with `null`, but I can't find a situation where `after` is defined 🤷‍♂️ 


<img width="644" alt="Screenshot 3" src="https://github.com/refined-github/refined-github/assets/1402241/0b46713a-a295-49c2-8a86-455d9e8ca4eb">
<img width="514" alt="Screenshot 2" src="https://github.com/refined-github/refined-github/assets/1402241/5a40a1a3-969c-4eb3-9655-b03bb5498b97">